config/default/user.role.agency_administrator.yml

```diff
@@ -34,17 +47,69 @@ permissions:
   - 'edit any agency_component content'
   - 'edit any webform'
   - 'edit field_agency'
+  - 'edit field_complex_average_days'
+  - 'edit field_complex_highest_days'
+  - 'edit field_complex_lowest_days'
+  - 'edit field_complex_median_days'
+  - 'edit field_expedited_average_days'
+  - 'edit field_expedited_highest_days'
+  - 'edit field_expedited_lowest_days'
+  - 'edit field_expedited_median_days'
+  - 'edit field_request_data_year'
+  - 'edit field_simple_average_days'
+  - 'edit field_simple_highest_days'
+  - 'edit field_simple_lowest_days'
+  - 'edit field_simple_median_days'
   - 'edit foia personnel entities'
   - 'edit own agency_component content'
   - 'edit own field_agency'
+  - 'edit own field_complex_average_days'
+  - 'edit own field_complex_highest_days'
+  - 'edit own field_complex_lowest_days'
+  - 'edit own field_complex_median_days'
+  - 'edit own field_expedited_average_days'
+  - 'edit own field_expedited_highest_days'
+  - 'edit own field_expedited_lowest_days'
+  - 'edit own field_expedited_median_days'
+  - 'edit own field_request_data_year'
+  - 'edit own field_simple_average_days'
+  - 'edit own field_simple_highest_days'
+  - 'edit own field_simple_lowest_days'
+  - 'edit own field_simple_median_days'
```

Comment:
Sorry @dangur, by 'non-admins' in [this comment](https://github.com/usdoj/foia-api/pull/93#discussion_r145154008), I meant everyone except the default Drupal administrator role. So even agency administrators should only be able to view these fields. These fields contain historical data, so we don't want to let anyone mess with it